components/sup/src/manager/mod.rs

```diff
@@ -1106,11 +1122,41 @@ impl Manager {
                 let mut specs = Self::generate_new_specs_from_package(&installed, &opts)?;
 
                 for spec in specs.iter_mut() {
-                    Self::save_spec_for(&mgr.cfg, spec)?;
-                    req.info(format!(
-                        "The {} service was successfully loaded",
-                        spec.ident
-                    ))?;
+                    match opts.render_only {
+                        None => {
+                            Self::save_spec_for(&mgr.cfg, spec)?;
+                            req.info(format!(
+                                "The {} service was successfully loaded",
+                                spec.ident
+                            ))?;
+                        }
+                        Some(_) => {
```

Comment:
By using `Some(_)` it seems like we'd be open to potentially confusing scenarios where `Some(false)` behaves as though it were `true`.